### PR TITLE
[SYCLomatic] Support migration of get VIRTUAL_MEMORY_MANAGEMENT_SUPPORTED attribute.

### PR DESCRIPTION
**Pull request by tangjj11**
Signed-off-by: Tang, Jiajun jiajun.tang@intel.com

### CODE REVIEW

**Review by intwanghao — APPROVED** [review]
LGTM

File: clang/runtime/dpct-rt/include/dpct/device.hpp [hunk 1]
```diff
@@ -512,6 +512,10 @@ class device_ext : public sycl::device {
     return get_device_info().get_global_mem_size();
   }
 
+  bool get_virtual_mem_support() const {
```
Comment (zhimingwang36) [inline clang/runtime/dpct-rt/include/dpct/device.hpp]: here don't need this help function.  Use has() directory.

Comment (tangjj11) [reply clang/runtime/dpct-rt/include/dpct/device.hpp]: fixed

**Review by zhimingwang36 — APPROVED** [review]

File: clang/test/dpct/driver_device_after11.2.cu [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+// RUN: dpct --format-range=none -out-root %T/driver_device_after11.2 %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/driver_device_after11.2/driver_device_after11.2.dp.cpp
+// RUN: %if build_lit %{icpx -c -fsycl %T/driver_device_after11.2/driver_device_after11.2.dp.cpp -o %T/driver_device_after11.2/driver_device_after11.2.dp.o %}
+
+#include <cuda.h>
+
+int main() {
+  int i;
+  CUdevice d;
+  // CHECK: i = dpct::get_device(d).has(sycl::aspect::ext_oneapi_virtual_mem);
```
Comment (zhimingwang36) [inline clang/test/dpct/driver_device_after11.2.cu:12]: Please further update the PR by introducing option to control the use of exp feature.

**Pull request merged**